content/open-source/wolfi/building-a-wolfi-package.md

```diff
@@ -260,41 +260,73 @@ update:
     strip-prefix: v
     tag-filter: v
 ```
-Again, this section is only required when submitting the package to Wolfi.
+Again, this section is only required when submitting the package to Wolfi. For more details about Wolfi's automated package updates, check [the official docs](https://github.com/wolfi-dev/os/blob/main/docs/UPDATES.md) on the subject.
 
-## Building the Packages
-When you feel your YAML is good to run, it's time to build the packages with melange. In this guide we'll use Docker to execute melange in a local environment. The procedure is explained in more detail in our [Getting Started with melange](https://edu.chainguard.dev/open-source/melange/tutorials/getting-started-with-melange/) tutorial.
+## Building Packages
+When you feel your YAML is good for a first run, it's time to build the package with melange. In this guide we'll use Docker to execute melange in a local environment, using [Wolfi's SDK](https://github.com/wolfi-dev/tools/pkgs/container/sdk) image. This image contains everything you need to build Wolfi packages with melange and Wolfi-based images with apko.
 
-First, you'll need to generate a melange key. This is required to sign the apks, and the same key will need to be used when installing these packages in an image with apko.
+The procedure to build apk packages with melange is explained in more detail in our [Getting Started with melange](https://edu.chainguard.dev/open-source/melange/tutorials/getting-started-with-melange/) tutorial.
 
-Make sure you have an updated version of the melange image:
+### Setting Up a Local Development Environment
+
+Start by cloning the [Wolfi-os](https://github.com/wolfi-dev/os) repository to your local machine. If you plan on sending a pull request to Wolfi later, you may want to create a fork now and clone your fork instead.
 
 ```shell
-docker pull cgr.dev/chainguard/melange
+git clone https://github.com/wolfi-dev/os.git
 ```
 
-Then, run the following command to generate the key:
+From the root of the project, run the following command to build your Docker-based development environment:
 
 ```shell
-docker run --rm -v "${PWD}":/work cgr.dev/chainguard/melange keygen
+make dev-container
+```
+
+This will create an ephemeral container based on the Wolfi SDK image, with a few predefined settings. We'll call this your **Wolfi development environment**.
+
+```
+❯ make dev-container
+docker run --privileged --rm -it \
+    -v "/home/erika/Projects/os:/home/erika/Projects/os" \
+    -w "/home/erika/Projects/os" \
+    -e SOURCE_DATE_EPOCH=0 \
```

Comment:
I wonder if we should (I've thought about it) just change the target to also specify:
```
-e BUILDWORLD=no \
```

here. Of course it wouldn't be here, but in wolfi. If we don't do that, I think you might need to specify on L307 below BUILDWORLD=no make package/<your-package-name>

Reply:
Just got confirmation from @imjasonh that this flag currently doesn't do anything, it was removed recently so probably why it's not documented in the repository docs.